### PR TITLE
[ Release ] NNTrainer 0.2.0 Release

### PR DESCRIPTION
**Pull request by jijoongmoon**
NNTrainer v1.0.0 is released.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,4 +1,10 @@
-nntrainer (0.1.1) UNRELEASED; urgency=medium
+nntrainer (1.0.0) unstable; urgency=medium
+
+  * 1.0.0 release
+
+ -- jijoongmoon <jijoongmoon@jijoong-DeskTop>  Wed, 02 Jun 2021 22:14:52 +0900
```
Comment (kparichay) [inline debian/changelog:5]: update email to samsung.com?

File: packaging/nntrainer.spec [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
 Name:		nntrainer
 Summary:	Software framework for training neural networks
-Version:	0.1.1
+Version:	1.0.0
```
Comment (kparichay) [inline packaging/nntrainer.spec:48]: i wonder how about release 1.0 once devel headers are a bit finalized? and release 0.2.0 for now?

File: debian/changelog (hunk 1 above)
Comment (jijoongmoon) [reply debian/changelog]: my bad. fixed

File: packaging/nntrainer.spec (hunk 2 above)
Comment (jijoongmoon) [reply packaging/nntrainer.spec]: then.. we will do 1.0.0 which includes.
  . finalize devel headers
  . recurrent net
  . tflite model support
  . and more.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

**Pull request merged**